cmd/pretty_pipe/README.md

```diff
@@ -0,0 +1,81 @@
+# Zerolog Pretty Pipe
```

Comment:
What about calling this command prettylog instead?

Reply:
Agreed.